pkgs/data/icons/catppuccin-cursors/default.nix

```
@@ -61,7 +46,7 @@ stdenvNoCC.mkDerivation {
         local variant=$(sed 's/\([A-Z]\)/-\1/g' <<< "$output")
         local variant=''${variant^}
 
-        cp -r dist/Catppuccin-"$variant"-Cursors "$iconsDir"
+        unzip cursors/Catppuccin-"$variant"-Cursors.zip -d "$iconsDir"
```

Maybe this would be better (or at least cleaner) as:
```suggestion
        unzip "cursors/Catppuccin-$variant-Cursors.zip" -d "$iconsDir"
```

cc @PlayerNameHere who wrote it this way originally